apis/python/src/tiledbsoma/_query.py

```diff
@@ -852,13 +856,16 @@ def _reindex(batch: pa.RecordBatch) -> pa.RecordBatch:
 
     approx_X_shape = tuple(b - a + 1 for a, b in matrix.non_empty_domain())
     # heuristically derived number (benchmarking). Thesis is that this is roughly 80% of a 1 GiB io buffer,
-    # which is the default for SOMA.
+    # which is the default for SOMA. If we have NNZ, use to partition based upon memory size. If we do not
```

Comment:
```suggestion
    # which is the default for SOMA. If we have fast NNZ, use to partition based upon memory size. If we do not
```

Sorry I didn't catch both spots earlier! :|